examples/website.py

```
@@ -0,0 +1,80 @@
+
+    username = 'admin'
+    password = '12345'
+
+    @paradise
+    def login(self):
+        print("* Enter your username:")
+        username = input("  ")
+        print("* Enter your password:")
+        password = input("  ")
+
+        return True if (username, password) == (self.username, self.password) else False
```

Ruff (0.8.2)

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)